components/microsoft_onedrive/actions/get-table/get-table.mjs

```diff
@@ -0,0 +1,92 @@
+import axios from "@pipedream/platform";
+
+export default {
+  name: "Get Table",
+  description: "Retrieve a table from an Excel spreadsheet stored in OneDrive",
+  key: "onedrive-get-excel-table",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    microsoft_onedrive: {
+      type: "app",
+      app: "microsoft_onedrive",
+      label: "",
+      description: "",
+    },

```

Comment:
We just `import onedrive from "../../microsoft_onedrive.app.mjs";`
and reference the app like this:

```suggestion
    onedrive,
```